x-pack/plugins/maps/public/routing/routes/maps_app/maps_app.js

```diff
@@ -0,0 +1,511 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import 'mapbox-gl/dist/mapbox-gl.css';
+import _ from 'lodash';
+import { DEFAULT_IS_LAYER_TOC_OPEN, FLYOUT_STATE } from '../../../reducers/ui';
+import {
+  getIndexPatternService,
+  getMapsCapabilities,
+  getToasts,
+  getData,
+  getUiSettings,
+  getCoreChrome,
+} from '../../../kibana_services';
+import { copyPersistentState } from '../../../reducers/util';
+import { getInitialLayers } from '../../bootstrap/get_initial_layers';
+import rison from 'rison-node';
+import { getInitialTimeFilters } from '../../bootstrap/get_initial_time_filters';
+import { getInitialRefreshConfig } from '../../bootstrap/get_initial_refresh_config';
+import { getInitialQuery } from '../../bootstrap/get_initial_query';
+import { getMapsSavedObjectLoader } from '../../bootstrap/services/gis_map_saved_object_loader';
+import { MapsTopNavMenu } from '../../page_elements/top_nav_menu';
+import {
+  getGlobalState,
+  updateGlobalState,
+  useGlobalStateSyncing,
+} from '../../state_syncing/global_sync';
+import { AppStateManager } from '../../state_syncing/app_state_manager';
+import { useAppStateSyncing } from '../../state_syncing/app_sync';
+import { getStoreSyncSubscription } from '../../store_operations';
+import { MapsRoot } from '../../page_elements/maps_root';
+import { updateBreadcrumbs } from '../../page_elements/breadcrumbs';
+import { esFilters } from '../../../../../../../src/plugins/data/public';
+
+export const MapsAppView = class extends React.Component {
```

Comment:
precede private methods in this class with `_`. (e.g. onQueryChange, ahndleStoreChanges,  etc..)